src/modules/Servicehosting/html_client/mod_servicehosting_manage.html.twig

```diff
@@ -80,7 +80,7 @@
             </div>
             {# Hosting password #}
             <div class="tab-pane fade" role="tabpanel" id="password">
-                <h3>{{ 'Change your FTP/cPanel/SSH password.'|trans }}</h3>
+                <h3>{{ 'Change your FTP/panel/SSH password'|trans }}</h3>
```

Comment:
I would probably write out "control panel" rather than just panel.
Other than that, these changes LGTM. Thanks!

Reply:
Works for me! I changed it to `control panel` and moved it to the beginning of the list; alphabetical ordering looks better to me, though I can of course change it back if needed :)